civicase.php

```diff
@@ -573,6 +573,14 @@ function civicase_civicrm_selectWhereClause($entity, &$clauses) {
   if ($entity === 'Case') {
     unset($clauses['id']);
   }
```

Comment:
Can we move this logic to the new hook since it concerns the Case entity too? Or what do you think?

Reply:
yes we should move it to a separate hook but I think we should do that in a separate ticket